src_py/sprite.py

```diff
@@ -1332,6 +1333,22 @@ def set_timing_treshold(self, time_ms):
 
         Raises TypeError if time_ms is not int or float.
 
+        """
+        warn("This function will be removed, use set_timing_threshold function instead")
```

Comment:
Ah, I just realised that `DeprecationWarning` must be passed as a second argument for this function for the warning to be a deprecation warning

Reply:
Done. Thanks